content/docs/tasks/traffic-management/secure-ingress-sds/index.md

```diff
@@ -0,0 +1,488 @@
+      - "helloworld-v1.example.com"
+      gateways:
+      - mygateway
+      http:
+      - match:
+        - uri:
+            exact: /hello
+        route:
+        - destination:
+            host: helloworld-v1
+            port:
+              number: 5000
+    EOF
+    {{< /text >}}
+
+1. Send HTTPS request to `helloworld-v1.example.com`
+
+    {{< text bash >}}
+    $ curl -v -HHost:helloworld-v1.example.com --resolve helloworld-v1.example.com:$SECURE_INGRESS_PORT:$INGRESS_HOST --cacert helloworld-v1.example.com/2_intermediate/certs/ca-chain.cert.pem https://helloworld-v1.example.com:$SECURE_INGRESS_PORT/hello
+    HTTP/2 200
+    {{< /text >}}
+
+1. Send HTTPS request to `httpbin.example.com` still returns teapot.
+
+    {{< text bash >}}
+    $ curl -v -HHost:httpbin.example.com --resolve httpbin.example.com:$SECURE_INGRESS_PORT:$INGRESS_HOST --cacert httpbin.example.com/2_intermediate/certs/ca-chain.cert.pem https://httpbin.example.com:$SECURE_INGRESS_PORT/status/418
+        -=[ teapot ]=-
+
+           _...._
+         .'  _ _ `.
+        | ."` ^ `". _,
+        \_;`"---"`|//
+          |       ;/
+          \_     _/
+            `"""`
+    {{< /text >}}
+
+## Configure a mutual TLS ingress gateway with secret discovery service enabled
+
+In this section you will extend your gateway's definition to support [mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication).
+We can change the credentials at ingress gateway by deleting the secret and creating a new one.
+This time we need to pass CA certificate that the server will use to verify its clients, and we must use the name `cacert` to hold CA certificate.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system delete secret httpbin-credential
+    $ kubectl create -n istio-system secret generic httpbin-credential  \
+    --from-file=key=httpbin.example.com/3_application/private/httpbin.example.com.key.pem \
+    --from-file=cert=httpbin.example.com/3_application/certs/httpbin.example.com.cert.pem \
+    --from-file=cacert=httpbin.example.com/2_intermediate/certs/ca-chain.cert.pem
+    {{< /text >}}
+
+1. Redefine Gateway and change the TLS mode to MUTUAL
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+     name: mygateway
+    spec:
+     selector:
+       istio: ingressgateway # use istio default ingress gateway
+     servers:
+     - port:
+         number: 443
+         name: https
+         protocol: HTTPS
+       tls:
+         mode: MUTUAL
+         serverCertificate: "use sds"         # arbitrary non-empty string
+         privateKey: "use sds"                # arbitrary non-empty string
+         caCertificates: "use sds"            # arbitrary non-empty string
+         credentialName: "httpbin-credential" # must be the same as secret
+       hosts:
+       - "httpbin.example.com"
+    EOF
+    {{< /text >}}
+
+1. Sending HTTPS request with previous way will fail
```

Comment:
Sending an HTTPS request using the prior approach now fails

Reply:
Done.